ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/ethernet-interfaces.j2

```diff
@@ -18,7 +18,7 @@ ethernet_interfaces:
 {%             for adapter_switch in adapter.switches %}
 {%                 if adapter_switch == inventory_hostname %}
 {%                     set channel_group_id = (adapter.switch_ports[0] | regex_findall("\d") | join) %}
-{%                     set peer_interface = endpoint_ports[loop.index0] %}
+{%                     set peer_interface = endpoint_ports[loop.index0] | arista.avd.default('N/A') %}
```

Comment:
This will show up in description as well, so I suggest just using an empty string as the default value.